modules/EvseSlac/main/evse_fsm.hpp

```diff
@@ -96,6 +97,9 @@ class EvseFSM {
 
     explicit EvseFSM(SlacIO& slac_io, int _set_key_timeout);
 
+    sigslot::signal<std::string> signal_ev_mac_address_match_cnf;
```

Comment:
function callbacks should be also fine instead of sigslot here, but hopefully with the new fsm lib, there is a better way to communicate to the outside via the fsm

Reply:
basically the signals are also just registering callback handlers